xmipp

```diff
@@ -196,6 +196,19 @@ def getSources(branch='', doPull=True):
         print(red("Cannot get the sources"))
     return ok
 
+def getDependencies():
+    print("Getting Dependencies -------------------------------------")
+    ok=True
+    advisorDir = "src/cuFFTAdvisor/" 
+    ok = checkProgram("git")
+    if ok and not os.path.exists(advisorDir):
+        createDir(advisorDir)
+        ok=runJob("git clone https://github.com/DStrelak/cuFFTAdvisor.git " + advisorDir)
```

Comment:
Why this? We realy need a new repo??

Reply:
well, this library is for performance optimization of the cuFFT library. Theoretically we can add the resulting shared library into xmipp repo, but I think that this way it's more clean

Reply:
Ok!